local_settings.sh.in

```diff
@@ -303,8 +303,34 @@ uninstall_policies() {
 	_restore_file_modes
 }
 
+verify_policies() {
+	if [[ ! selinuxenabled ]]; then
+		echo "SELinux is disabled"
+		exit 0
+	fi
+	if [ $(getenforce) != Enforcing ]; then
+		echo "SELinux is permissive";
+		exit 0
+	fi
+
+	failed_count=0
+	for module in $MODULES; do
+		local_failed=1
+		while read installed_module; do
+			if [ "$module" == "$installed_module" ]; then
+				local_failed=0
+				break
+			fi
+		done < <(semodule -l)
+		test ${local_failed} -ne 0 && echo "Missing ${module}!"
+		let "failed_count+=$local_failed"
+	done
+	echo "Found ${failed_count} missing module(s)."
+	(test ${failed_count} -eq 0 && exit 0) || exit 1
```

Comment:
I'd just do:

[ $failed_count -eq 0 ]
exit $?

or (same thing):

test $failed_count -eq 0
exit $?

Reply:
indeed. the parenthesis thing is a mess - better avoiding it. And "exit true if true" is... ;).